app-web/src/components/ComponentPreview/ComponentPreview.js

```diff
@@ -28,54 +28,55 @@ export default class ComponentPreview extends React.Component {
     };
   }
 
-  async componentDidMount() {
+    async componentDidMount() {
 
-    //pull in the values to retrieve to preview content from GitHub from props, if provided
-    let { owner, repo, path, branch } = this.props;
+        //pull in the values to retrieve to preview content from GitHub from props, if provided
+        let {owner, repo, path, branch} = this.props;
 
-    //allow for default values - relative to file that has embedded the component -  if values for props not provided
-    owner =  owner || this.props.node.source._properties.owner;
-    repo = repo || this.props.node.source._properties.repo;
-    branch = branch || this.props.node.source._properties.branch;
+        //allow for default values - relative to file that has embedded the component -  if values for props not provided
+        owner = owner || this.props.node.source._properties.owner;
```

Comment:
Can we document what properties are passable/are expected to this component?
The React PropTypes would be a great use case for that here.

Reply:
thanks for the suggestion. I've added PropTypes to the component to provide validation and some minimal doco.  Room for improvement on the doco side for sure.